src/Integration/Symfony/Resources/config/services.yaml

```diff
@@ -12,4 +12,3 @@ services:
     biig_domain.instantiator.default:
         alias: Biig\Component\Domain\Model\Instantiator\Instantiator
         public: true
-
```

Comment:
Need new line at the end.

Reply:
Hmm, I don't get it, this empty line exists in my file, why is it pushed this way ? Any idea ?

Reply:
I pushed again with one more empty line : seems to resolve the problem ...